client/admin/apps/AppDetailsPage.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useCallback } from 'react';
+import { Button, ButtonGroup, Icon, Avatar, Box, Divider, Chip, Margins, Skeleton } from '@rocket.chat/fuselage';
+
+import Page from '../../components/basic/Page';
+import { useRoute } from '../../contexts/RouterContext';
+import PriceDisplay from './PriceDisplay';
+import { AppStatus } from './AppStatus';
+import { useTranslation } from '../../contexts/TranslationContext';
+import { useAppInfo } from './hooks/useAppInfo';
+import { AppMenu } from './AppMenu';
+import { useLoggedInCloud } from './hooks/useLoggedInCloud';
+
+const objectFit = { objectFit: 'contain' };
+
+function AppDetailsPageContent({ data, setModal, isLoggedIn }) {
+	const t = useTranslation();
+
+	const {
+		iconFileData = '',
+		name,
+		author: { name: authorName, homepage, support } = {},
+		description,
+		categories = [],
+		version,
+		price,
+		purchaseType,
+		pricingPlans,
+		iconFileContent,
+		installed,
+		bundledIn,
+	} = data;
+
+
+	return <><Box display='flex' flexDirection='row' mbe='x20' w='full'>
+		<Avatar style={objectFit} size='x120' mie='x20' url={iconFileContent || `data:image/png;base64,${ iconFileData }`}/>
+		<Box display='flex' flexDirection='column' justifyContent='space-between' flexGrow={1}>
+			<Box fontScale='h1' fontWeight='500'>{name}</Box>
```

Comment:
Headings should fit in some font scale, without additional changes. Otherwise, we need to discuss if a new heading style is needed.